tslint.json

```diff
@@ -12,6 +12,10 @@
     "no-void-expression": false,
     "no-redundant-jsdoc": false,
     "prefer-type-cast": false,
+    "interface-name": false,
+    "no-multiline-string": false,
+    "no-require-imports": false,
+    "strict-boolean-expressions": false,
```

Comment:
Could I get your reasoning for disabling this? I like `strict-boolean-expressions` because truthy in JavaScript can be confusing sometimes.

Reply:
My preference isn't too strong, I can change this back. I mostly like it for "presentness" tests (is something either null or undefined) but it's not a huge issue, and you're right, it is confusing